src/microparams-init.c

```diff
@@ -930,6 +930,45 @@ void xnn_init_qs8_to_qs8_qc8w_scale_fp32_params(
   }
 }
 
+void xnn_init_qs8_qb8w_scale_fp32_params(
```

Comment:
should this be named qb4w?

Reply:
Though you may be coming from consistency point of view with-in qb*w. I would argue that, it shouldn't matter actually because the routine is independent of weight bit width. If you [look](https://github.com/google/XNNPACK/blob/master/src/operators/fully-connected-nc.c#L521) qc4w, it uses qc8w routine. I am not sure if we will ever write qb8w but this is just an artifact from qc.

Reply:
Sounds good. Let me update.